home.php

```diff
@@ -14,27 +24,37 @@
          </div>-->
 		  
          <div class = "nav">
+			 
+			 <div class = "logo" style = float:right>
+				
+				<center>
+						
+					<p>logo</p>
+					
+				</center>
+				
+			 </div>
+			 
+			 <div class = "admin_nav" style = float:right>
+				 <?php
```

Comment:
Added the logo div class further up the page and added the admin only session code